website/LICENSE.md

```diff
@@ -0,0 +1,24 @@
+# The MIT License (MIT)
+
+Copyright © `2020` `The Sapphire Community and its contributors`
```

Comment:
Is it intended that you don't plan on sublicensing?

Reply:
Yeah, doesn't really matter